src/modules/main/store/mainSlice.ts

```diff
@@ -21,7 +22,7 @@ interface INewestPostPayload {
 
 export interface IMainState {
   newest: INewestPostPayload;
-  important: IPost[];
+  important: PostResponseType[];
```

Comment:
should be still IPost[],
you can align IPost to have the same properties as on the response

Reply:
done